### PR TITLE
Update about-access.md

### PR DESCRIPTION
**Pull request by jeremyyeo**
## Checklist

- [ ] Review the [Content style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/content-style-guide.md) and [About versioning](https://github.com/dbt-labs/docs.getdbt.com/blob/current/contributing/single-sourcing-content.md#adding-a-new-version) so my content adheres to these guidelines.
- [ ] Add a checklist item for anything that needs to happen before this PR is merged, such as "needs technical review" or "change base branch."

Adding new pages (delete if not applicable):
- [ ] Add page to `website/sidebars.js`
- [ ] Provide a unique filename for the new page

Removing or renaming existing pages (delete if not applicable):
- [ ] Remove page from `website/sidebars.js`
- [ ] Add an entry `_redirects`
- [ ] [Ran link testing](https://github.com/dbt-labs/docs.getdbt.com#running-the-cypress-tests-locally) to update the links that point to the deleted page

### CODE REVIEW

File: website/docs/docs/collaborate/manage-access/about-access.md [hunk 1]
```diff
@@ -148,3 +148,7 @@ sign-in time based on the user's IdP-provided group membership information.
 - **Can I configure a user's License Type based on IdP Attributes?** Yes, see
   the docs on [managing license types](/cloud-seats-and-users#managing-license-types)
   for more information.
+
+- **Why can't I edit the group membership of a particular user?** Check to see that
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/manage-access/about-access.md:152]:
```suggestion
- **Why can't I edit a user's group membership?** Make sure you're not trying to edit your own user as this isn't allowed for security reasons. To edit the group membership of your own user, you'll need a different user to make those changes.
```

File: website/docs/docs/collaborate/manage-access/about-access.md [hunk 2]
```diff
@@ -136,15 +136,14 @@ sign-in time based on the user's IdP-provided group membership information.
...
+  Group memberships are updated every time a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud for these group memberships to be synchronized.
+
+- **Can I set up SSO without RBAC?**  <br />
+  Yes, see the documentation on [Manual Assignment](#manual-assignment) above for more information on using SSO without RBAC.
+
+- **Can I configure a user's License Type based on IdP Attributes?** <br />
+  Yes, see the docs on [managing license types](/cloud-seats-and-users#managing-license-types) for more information.
+
+- **Why can't I edit the group membership of a particular user?**  <br />
+  Check to see that you are not trying to edit the group memberships of your own user as this is not allowed. You will need a different user to modify your own users group memberships.
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/manage-access/about-access.md]:
```suggestion
Make sure you're not trying to edit your own user as this isn't allowed for security reasons. To edit the group membership of your own user, you'll need a different user to make those changes.
```

File: website/docs/docs/collaborate/manage-access/about-access.md [hunk 3]
```diff
@@ -136,15 +136,14 @@ sign-in time based on the user's IdP-provided group membership information.
...
+- **When are IdP group memberships updated for SSO Mapped groups?**  <br />
+  Group memberships are updated every time a user logs into dbt Cloud via a supported SSO provider. If you've changed group memberships in your identity provider or dbt Cloud, ask your users to log back into dbt Cloud for these group memberships to be synchronized.
+
+- **Can I set up SSO without RBAC?**  <br />
+  Yes, see the documentation on [Manual Assignment](#manual-assignment) above for more information on using SSO without RBAC.
+
+- **Can I configure a user's License Type based on IdP Attributes?** <br />
+  Yes, see the docs on [managing license types](/cloud-seats-and-users#managing-license-types) for more information.
+
+- **Why can't I edit the group membership of a particular user?**  <br />
```
Comment (mirnawong1) [inline website/docs/docs/collaborate/manage-access/about-access.md]:
```suggestion
- **Why can't I edit a user's group membership?**  <br />
```

**Comment by mirnawong1** [comment]
hi @jeremyyeo thanks for opening this up to clarify the faqs. made some small suggestions for your review

**Review by jeremyyeo — COMMENTED** [review]
LGTM :) We can approve and merge :)

**Review by mirnawong1 — APPROVED** [review]
merging this @jeremyyeo !

**Pull request merged**